src/Parser/Druid/Balance/Modules/Spells/MoonSpells.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatPercentage } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class MoonSpells extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    abilityTracker: AbilityTracker,
+  };
+  get availableCasts() {
+    const hasRM = this.combatants.selected.hasBack(ITEMS.RADIANT_MOONLIGHT.id);
+    const cooldown = hasRM ? 7.5 : 15;
+    const totalFromCD = Math.floor(this.owner.fightDuration / 1000 / cooldown);
+    const totalFromPull = 3;
+    return totalFromCD + totalFromPull;
+  }
+
+  get totalCasts() {
+    return this.abilityTracker.getAbility(SPELLS.NEW_MOON.id).casts 
+         + this.abilityTracker.getAbility(SPELLS.HALF_MOON.id).casts
+         + this.abilityTracker.getAbility(SPELLS.FULL_MOON.id).casts;
+  }
+
+  suggestions(when) {   
+    const casted = this.totalCasts;
+    const percCasted = casted / this.availableCasts;
+
+    when(percCasted).isLessThan(1)
```

Comment:
While it's not necessary, defining a `suggestionThresholds` getter might save you some time refactoring once you get to implementing the checklist.